pol-core/bscript/compiler/codegen/CodeGenerator.h

```diff
@@ -18,9 +17,13 @@ class CodeGenerator
                                                    const LegacyFunctionOrder* );
 
 private:
-  CodeGenerator();
+  explicit CodeGenerator( InstructionEmitter& );
 
   void generate_instructions( CompilerWorkspace& );
+
+private:
+  InstructionEmitter& emitter;
+  InstructionEmitter& emit;
```

Comment:
There are two of these because sometimes when calling a method on `InstructionEmitter`, the variable name reads better as a noun, and sometimes it reads better as a verb.

[Example.](https://github.com/polserver/polserver/blob/ecompile-antlr/pol-core/bscript/compiler/codegen/InstructionGenerator.cpp#L667)